app/controllers/reports/regions_controller.rb

```diff
@@ -195,6 +211,8 @@ def find_region
 
   def region_class
     @region_class ||= case report_params[:report_scope]
+    when "state"
```

Comment:
I _think_ we don't use this magical `region_class` as a class anywhere 🤔  That means this method is doing some trivial mapping, plus an inclusion check.

Can we: 
1. trash this method
1. `switch` on the `report_scope` in `find_region`
1. Rely on the inclusion check there?

Reply:
yup, I'm going to do this in a followup PR because I have a slightly larger refactoring in mind here.